disco_aws_automation/disco_elb.py

```diff
@@ -56,9 +56,12 @@ def get_cname(self, hostclass, domain_name):
     def _setup_health_check(self, elb_name, health_check_url, instance_protocol, instance_port):
         if not health_check_url:
             logging.warning("No health check url configured for ELB %s", elb_name)
-            return
+            if instance_protocol in ('HTTP', 'HTTPS'):
```

Comment:
Should probably make `instance_protocol` uppercase with `.upper()`, just to be sure.